TrafficCapture/dockerSolution/src/main/docker/docker-compose-multi.yml

```diff
@@ -14,8 +14,9 @@ services:
       - cluster.name=es-cluster
       - discovery.seed_hosts=capture-proxy-es2
       - cluster.initial_master_nodes=capture-proxy-es1,capture-proxy-es2
+      - discovery.type=zen
```

Comment:
We should probably add a comment, that different versions of elasticsearch might not use this discovery type and use "multi-node"